src/upcoming-components/Textarea/Textarea.module.css

```diff
@@ -1,13 +0,0 @@
-@import '../../design-tokens/mixins.css';
-
-/*------------------------------------*\
-    #TEXTAREA
-\*------------------------------------*/
-
-/**
- * Inherits base textarea styles 
- */
-.textarea {
-  @mixin inputStyles;
```

Comment:
This is probably valuable part of the textarea component, where the `inputStyles` mixin is reusable and keeps in line with theming of the inputfield.